x-pack/plugin/sql/qa/server/src/main/java/org/elasticsearch/xpack/sql/qa/rest/RestSqlUsageTestCase.java

```diff
@@ -224,6 +240,47 @@ public void testSqlRestUsage() throws IOException {
         assertClientTypeAndAllQueryMetrics(clientTypeTotalQueries, allTotalQueries, responseAsMap);
     }
 
+    public void testScrollUsage() throws IOException {
+        index(testData);
+
+        String cursor = runSql("SELECT page_count, name FROM library ORDER BY page_count", 1);
```

Comment:
You could randomize the `fetchSize` value with something between 1 and `testData.size()`.